include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -24,10 +24,228 @@
 #    include "hetero/algorithm_impl_hetero.h" //TODO: for __brick_copy
 #endif
 
+#if _ONEDPL_CPP20_RANGES_PRESENT
+#    include <ranges>
+#    include <functional>
+#    include "algorithm_ranges_impl.h"
+#endif
+
 namespace oneapi
 {
 namespace dpl
 {
+namespace ext
```

Comment:
1. We are introducing `oneapi::dpl::ext::ranges` due to parallel ranges algorithms are not a part of C++ standard library yet.
when is standardized we will move the code to  `oneapi::dpl::ranges` (a core feature).
2. We came to an agreement that we don't  need sperate  declarations `glue_algorithm_defs.h` for the functional objects.

Reply:
`for_each` declaration should be removed from `glue_algorithm_defs.h` then, should not it?

Reply:
`glue_algorithm_ranges_defs.h` is a part of "experimental"  ranges API. We keep it as was before. At least because it works with C++17 and has more range based algorithms with hetero policy.